README.md

```diff
@@ -1,5 +1,4 @@
-The FreeRTOS 202104.00 introduces new coreMQTT Agent library, and includes all the Long-Term Support (LTS) libraries that are part of the [FreeRTOS-LTS 202012.01-LTS](https://github.com/FreeRTOS/FreeRTOS-LTS/tree/202012.01-LTS) release. Learn more about the FreeRTOS 202012.01 LTS libraries at [https://freertos.org/lts-libraries.html](https://freertos.org/lts-libraries.html).
-
+The [FreeRTOS 202104.00](https://github.com/FreeRTOS/FreeRTOS/tree/202104.00) introduces new [coreMQTT Agent library](https://github.com/FreeRTOS/coreMQTT-Agent), and includes all the Long-Term Support (LTS) libraries that are part of the [FreeRTOS-LTS 202012.01-LTS](https://github.com/FreeRTOS/FreeRTOS-LTS/tree/202012.01-LTS) release. Learn more about the FreeRTOS 202012.01 LTS libraries at [https://freertos.org/lts-libraries.html](https://freertos.org/lts-libraries.html).
```

Comment:
```suggestion
The [FreeRTOS 202104.00](https://github.com/FreeRTOS/FreeRTOS/tree/202104.00) release introduces the new [coreMQTT Agent library](https://github.com/FreeRTOS/coreMQTT-Agent), and includes all the Long-Term Support (LTS) libraries that are part of the [FreeRTOS-LTS 202012.01-LTS](https://github.com/FreeRTOS/FreeRTOS-LTS/tree/202012.01-LTS) release. Learn more about the FreeRTOS 202012.01 LTS libraries at [https://freertos.org/lts-libraries.html](https://freertos.org/lts-libraries.html).
```

Reply:
Also,
> Long-Term Support (LTS) libraries that are part of the FreeRTOS-LTS 202012.01-LTS release

I think 3 LTS's is too many. It'd make sense to remove one
```
Long-Term Support (LTS) libraries that are part of the FreeRTOS 202012.01-LTS release
```

Reply:
FreeRTOS-LTS 202012.01-LTS represents the 202012.01-LTS tag in the FreeRTOS-LTS repository
https://github.com/FreeRTOS/FreeRTOS-LTS/tree/202012.01-LTS